Ratings_MVP/Ratings_MVP/MovieDetailsScreen/View/MovieDetailsViewController.swift

```diff
@@ -28,10 +29,9 @@ final class MovieDetailsViewController: UIViewController {
         return tableView
     }()
 
-    private let networkService = NetworkService()
-
-    // MARK: Private Properties
+    // MARK: - Public Properties
 
+    var presenter: MovieDetailsPresenterProtocol!
     var currnetMovie: Movie?
```

Comment:
```suggestion
    var currnetMovie: Movie?
```
Думаю, этого здесь быть не должно

Reply:
верно